### PR TITLE
Hr 20220608 002

### PR DESCRIPTION
**Pull request by 151henry151**

### CODE REVIEW

File: web/src/lib/components/Login.svelte [hunk 1]
```diff
@@ -42,7 +42,7 @@
 <div class="pt-10 flex justify-center items-center">
     {#if qr}
         <div>
-            <p class="mb-4">Scan with a <a class="link" href="https://github.com/fiatjaf/lnurl-rfc#lnurl-documents" target="_blank">compatible</a> wallet to enter the marketplace.</p>
+            <p class="mb-4 text-center">Scan with <a class="link" href="https://breez.technology/">Breez,</a> <a class="link" href="https://phoenix.acinq.co/">Phoenix,</a> <a class="link" href="https://thunderhub.io/">Thunderhub,</a> <a class="link" href="https://zeusln.app/">Zeus,</a> <a class="link" href="https://getalby.com/">Alby,</a> or any <a class="link" href="https://github.com/fiatjaf/lnurl-rfc#lnurl-documents" target="_blank">LNurl compatible</a> wallet to enter the marketplace.</p>
```
Comment (ibz) [inline web/src/lib/components/Login.svelte:45]: Can one "scan" with Thunderhub though? Isn't it a web app?

File: web/src/lib/components/Auction.svelte [hunk 2]
```diff
@@ -92,6 +93,7 @@
                 <div class="lg:w-1/3">
                     <h2 class="text-3xl text-center mb-4">{auction.title}</h2>
                     <Gallery photos={auction.media} />
```
Comment (ibz) [inline web/src/lib/components/Auction.svelte:95]: I would much rather prefer we show/hide the components where they are used rather than having the component know it should show only in some cases. That makes them more reusable.

**Review by ibz — CHANGES_REQUESTED** [review]

File: web/src/lib/components/Auction.svelte (hunk 2 above)
Comment (151henry151) [reply web/src/lib/components/Auction.svelte:96]: Agreed, that is much better, to keep things modular.

Comment (151henry151) [reply web/src/lib/components/Auction.svelte:96]: Fixed in [a0fbd21](https://github.com/PlebeianTech/plebeian-market/pull/12/commits/a0fbd21dbd8cb4798388020e7d391be0603ae620)

File: web/src/lib/components/Login.svelte (hunk 1 above)
Comment (151henry151) [reply web/src/lib/components/Login.svelte]: Addressed this with [befad84](https://github.com/PlebeianTech/plebeian-market/pull/12/commits/befad84d89ffda42e09a66ab0ae7335ba0b3adab) but we may want to look at it some more, it is getting a bit wordy this way.
